pool/app/waiting_list/repo/repo.ml

```diff
@@ -134,11 +135,39 @@ module Sql = struct
     |> Caqti_type.string ->* RepoEntity.Experiment.t
   ;;
 
-  let find_by_experiment pool id =
-    Utils.Database.collect
-      (Pool_database.Label.value pool)
-      find_by_experiment_request
-      (Experiment.Id.value id)
+  let select_count =
+    let select_from =
+      {sql|
+      SELECT COUNT(*)
+        FROM
+        pool_waiting_list
+      LEFT JOIN pool_contacts
+        ON pool_waiting_list.contact_id = pool_contacts.id
+      LEFT JOIN user_users
+        ON pool_contacts.user_uuid = user_users.uuid
+      |sql}
+    in
+    Format.asprintf "%s %s" select_from
```

Comment:
Same as above and inconsistent. We should check if we generally add a newline `\n` or just concatenate it.